modules/mpv-player/android/src/main/java/expo/modules/mpvplayer/nativeplayer/SystemVolumeController.kt

```
@@ -0,0 +1,113 @@
+
+    init {
+        registerReceiver()
+    }
+
+    private fun registerReceiver() {
+        if (receiverRegistered) return
+        try {
+            val filter = IntentFilter("android.media.VOLUME_CHANGED_ACTION")
+            context.registerReceiver(volumeReceiver, filter)
+            receiverRegistered = true
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)